Kubernetes/windows/debug/collectlogs.ps1

```diff
@@ -21,6 +21,7 @@ ipmo $helper
 DownloadFile -Url  "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/dumpVfpPolicies.ps1" -Destination $BaseDir\dumpVfpPolicies.ps1
 DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/hns.psm1" -Destination $BaseDir\hns.psm1
 DownloadFile -Url "https://raw.githubusercontent.com/$GithubSDNRepository/master/Kubernetes/windows/debug/starthnstrace.cmd" -Destination $BaseDir\starthnstrace.cmd
```

Comment:
Do we want to keep [starthnstrace.cmd](https://github.com/microsoft/SDN/blob/master/Kubernetes/windows/debug/starthnstrace.cmd), or replace it?

Reply:
I'm going to say "yes" for now.